src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![crate_name = "librespot"]
```

Comment:
Should the crate have the "lib"-prefix? I don't see any other crates with that prefix, and we currently end up with a liblibrespot.rlib binary.